tests/test_functons.py

```diff
@@ -112,3 +118,80 @@ def test_distance_from_zero(test_distance_from_zero_input, test_distance_from_ze
 ])
 def test_square(distance_from_zero_type_input, distance_from_zero_type_expected):
     assert distance_from_zero_type(distance_from_zero_type_input) == distance_from_zero_type_expected
```

Comment:
@nicolas13sochi ooops i made a mistake - it's a test function so it should be prepended with `test_` so ....name it `test_distance_from_zero_type()` please